<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FriendDataCard</class>
 <widget class="QWidget" name="FriendDataCard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>224</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>409</width>
    <height>224</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>409</width>
    <height>224</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lb_feeling">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>签名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_icon">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>头像</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_icon">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/tx/1.png</normaloff>:/tx/1.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lb_name">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>昵称：</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_setName">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>15</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>随便整个名</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_setFeeling">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>65</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>111</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_close">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>170</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_deleteFriend">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>170</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>删除好友</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
